luigi/scheduler.py

```diff
@@ -170,7 +170,8 @@ def _get_default(x, default):
 class Task(object):
 
     def __init__(self, task_id, status, deps, resources=None, priority=0, family='', module=None,
-                 params=None, disable_failures=None, disable_window=None, disable_hard_timeout=None):
+                 params=None, disable_failures=None, disable_window=None, disable_hard_timeout=None,
+                 runnable=False):
```

Comment:
If it's post-initialization (not in the __init__), why do you need it as a kwarg in the __init__? Can you give a line-hyperurl?